src/main/java/uk/gov/hmcts/reform/civil/service/CaseNoteService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.model.CaseNote;
+import uk.gov.hmcts.reform.civil.model.common.Element;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static java.util.Optional.ofNullable;
+import static uk.gov.hmcts.reform.civil.utils.ElementUtils.element;
+
+@Service
+@RequiredArgsConstructor
+public class CaseNoteService {
+
+    private final IdamClient idamClient;
+
+    public CaseNote buildCaseNote(String authorisation, String note) {
+        UserDetails userDetails = idamClient.getUserDetails(authorisation);
+
+        return  CaseNote.builder()
+            .createdBy(userDetails.getFullName())
```

Comment:
In your other PR for the CCD Definition you attached a screenshot (thanks for that) showing the screen in action. Just wondering how comes we see email addresses there as created by?
![image](https://user-images.githubusercontent.com/138029/125117123-a4af0c00-e0e5-11eb-8a08-9d0a7bdd43dd.png)

Reply:
Same behaviour as on master currrently for Author in history. Is this not normal?

<img width="1286" alt="Screenshot 2021-07-15 at 14 55 53" src="https://user-images.githubusercontent.com/8037782/125800219-bfb14092-4227-49bc-a94b-e8389e44986f.png">

Reply:
I hadn't even noticed that! Then if that's the case I guess we are fine. Thanks for double checking.